src/codewhisperer/commands/onInlineAcceptance.ts

```diff
@@ -59,7 +59,12 @@ export const acceptSuggestion = Commands.declare(
                     references,
                 },
                 context.extensionContext.globalState
-            )
+            ).finally(async () => {
+                if (isInlineCompletionEnabled()) {
```

Comment:
We probably don't need to check isInlineCompletionEnabled() here because this entire code path is under `isInlineCompletionEnabled`'s logical branch.